src/components/Quiz/QuizzesStats.tsx

```diff
@@ -0,0 +1,192 @@
+        {/* user stats */}
+        <Grid
+          gap={4}
+          bg="backgroundHighlight"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={4}
+          mb={-2}
+        >
+          <GridItem colSpan={{ base: 2, lg: 1 }} alignSelf="center" order={1}>
+            <Text
+              color="body"
+              fontWeight="bold"
+              fontSize="xl"
+              margin={0}
+              textAlign={{ base: "center", lg: "left" }}
+            >
+              <Translation id="your-total" />
+            </Text>
+          </GridItem>
+
+          <GridItem
+            colSpan={{ base: 2, lg: 1 }}
+            justifySelf={{ base: "auto", lg: "end" }}
+            alignSelf="center"
+            order={{ base: 3, lg: 2 }}
+          >
+            <Button
+              variant="outline-color"
+              leftIcon={<Icon as={FaTwitter} />}
+              onClick={() =>
+                handleShare({ score: userScore, total: TOTAL_QUIZZES_POINTS })
+              }
+              w={{ base: "full", lg: "auto" }}
+              mt={{ base: 2, lg: 0 }}
+            >
+              <Translation id="share-results" />
+            </Button>
+          </GridItem>
+
+          <GridItem colSpan={2} order={{ base: 2, lg: 3 }}>
+            <Stack gap={2}>
+              <Flex
+                justifyContent={{ base: "center", lg: "flex-start" }}
+                alignItems="center"
+              >
+                <Circle size="64px" bg="primary" mr={4}>
+                  <TrophyIcon color="neutral" w="35.62px" h="35.62px" />
+                </Circle>
+
+                <Text fontWeight="bold" fontSize="5xl" mb={0} color="body">
+                  {userScore}
+                  <Text as="span" color="bodyMedium">
+                    /{TOTAL_QUIZZES_POINTS}
+                  </Text>
+                </Text>
+              </Flex>
+
+              <Progress value={userScore} />
+
+              <Flex direction={{ base: "column", lg: "row" }}>
+                <Text mr={10} mb={0} mt={{ base: 2, lg: 0 }} color="bodyMedium">
+                  <Translation id="average-score" />{" "}
+                  <Text as="span" color="body">
+                    {parsedAverage}%
+                  </Text>
+                </Text>
+
+                <Text mb={0} color="bodyMedium">
+                  <Translation id="completed" />{" "}
+                  <Text as="span" color="body">
+                    {numberOfCompletedQuizzes}/{totalQuizzesNumber}
+                  </Text>
+                </Text>
+              </Flex>
+            </Stack>
+          </GridItem>
+        </Grid>
+
+        {/* community stats */}
+        <Flex
+          direction="column"
+          gap={6}
+          justifyContent="space-between"
+          bg="backgroundHighlight"
+          borderRadius={{ base: "none", lg: "lg" }}
+          border="none"
+          p={4}
```

Comment:
```suggestion
          p={8}
```

Fixes alignment

<img width="429" alt="Screenshot 2023-06-02 at 11 32 02" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/cc8bc0e4-4720-4b83-89be-169103bd0d77">
<img width="366" alt="Screenshot 2023-06-02 at 11 36 11" src="https://github.com/ethereum/ethereum-org-website/assets/62268199/4696e9cb-3e1d-4dda-be78-dbb564147d37">